test/reducers/dataReducer.test.js

```diff
@@ -84,3 +85,11 @@ test('data reducer EVENTS_LOAD_FAILED action', t => {
   t.equal(actual.error.message, 'Error: Something failed', 'sets error message by serialising the error')
   t.end()
 })
+
+test('data reducer HIDE_ERROR action', t => {
+  const expected = {errorVisible: false}
+  const actual = reducer({errorVisible: true},
```

Comment:
But it doesn't matter, if the error itself changes, important is that it's not visible, right?